cli/QuickstepCli.cpp

```diff
@@ -430,6 +445,13 @@ int main(int argc, char* argv[]) {
         reset_parser = true;
         break;
       }
+#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
+      // Profile only if profile_file_name flag is set
+      if (!started_profiling && !quickstep::FLAGS_profile_file_name.empty()) {
```

Comment:
Thanks for the reply.

Then, we should also put the explanation as a comment in the file, and we'll be good to merge. Thanks!